src/main/java/spoon/compiler/SpoonFile.java

```diff
@@ -54,8 +56,9 @@
 		byte[] bytes;
 		try {
 			InputStream contentStream = getContent();
```

Comment:
The contentStream must be closed. The best is:
```java
try (InputStream contentStream = getContent()) {
```

Reply:
Ooops, my bad, thanks.